src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -1165,7 +1165,9 @@ int CASE_TestSecurePairing_Setup(void * inContext)
 {
     chip::Platform::MemoryInit();
 
+#if !defined(CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK)
     chip::DeviceLayer::PlatformMgr().InitChipStack();
```

Comment:
Wait, why?  The rest of the test uses the platform manager, no?  How is it ok to use it without initializing it?

Reply:
We initialize the Chip stack from the application main level.

Reply:
Then this needs to be documented clearly here.  And presumably all other tests that have InitChipStack should have similar bits?

But that does raise the question: _why_ is that being initialized elsewhere instead of just per-test as needed?

Reply:
Fixed by improving the OIS platform manager. The changes were removed.